peachjam/templates/peachjam/saved_document/_folders_list.html

```diff
@@ -15,15 +15,54 @@ <h2>{% trans 'Saved documents' %}</h2>
         <h5 class="card-header">
           <i class="bi bi-folder"></i> {{ folder.name }}
         </h5>
+        <div class="card-header">
```

Comment:
You've got a duplicate header - move the `ul` up into the header above and it'll look better. That's how it's designed to work.

Also, we should only show this if peachjam_ml is configured. We can either do template inheritance with overridden blocks, or we can assume that `peachjam` knows about the existence of `peachjam_ml` (like it knows about `peachjam_search`) and have something like `settings.PEACHJAM["PEACHJAM_ML"] = True` that gets injected into templates.